core/loadpoint.go

```diff
@@ -317,6 +323,12 @@ func (lp *LoadPoint) configureChargerType(charger api.Charger) {
 		_ = lp.bus.Subscribe(evChargeStop, ct.StopCharge)
 		lp.chargeTimer = ct
 	}
+
+	// add Active timer for wakeup check
+	at := NewActiveTimer()
+	_ = lp.bus.Subscribe(evChargeCurrent, func(current float64) { at.ActiveTimerHandler(lp.enabled, current) })
+	_ = lp.bus.Subscribe(evChargeStop, func() { at.StopActiveTimer() })
```

Comment:
bin nicht sicher ob das Event immer rechtzeitig kommt

Reply:
hab es jetzt einfach in den loop mit eingebaut und aufgeräumt.